src/app/component/quiz/quiz.component.html

```diff
@@ -24,121 +21,132 @@
-                        type="button"
-                        (click)="next(i)"
-                        class="btn btn-success"
-                        aria-hidden="true"
-                        data-bs-slide="next"
-                        style="
-                          position: absolute;
-                          left: 73%;
+                                <label
+                                  class="form-check-label option"
+                                  *ngIf="card.type == 'radio'"
+                                  [ngClass]="
+                                    FormArray.controls.at(i)?.value
+                                      .radioValue === card.value
+                                      ? 'hightlight-selected'
+                                      : ''
+                                  "
+                                >
+                                  <input
+                                    type="radio"
+                                    class="form-check-input opt_check"
+                                    formControlName="radioValue"
+                                    value="{{ card.value }}"
+                                  />
+                                  {{ card.lable }}
+                                </label>
+                              </div>
+                            </ng-container>
+                          </div>
+                        </div>
+
+                        <div class="card-footer footer_txt">
+                          <div class="total_que">
+                            <p>
+                              <span>{{ i + 1 }}</span> Of
+                              <span>{{ question.length - 1 }}</span> Questions
+                            </p>
+                          </div>
 
-                          width: 20%;
-                        "
-                      >
-                        {{
-                          question.length != i+1
-                            ? "Next"
-                            : "Submit"
-                        }}
-                      </button>
-                      <button
-                        [disabled]="FormArray.controls[i].dirty"
-                        class="carousel-control-Previous"
-                        id="skip"
-                        role="button"
-                        data-slide="skip"
-                        type="button"
-                        (click)="skip(i)"
-                        class="btn btn-dark"
-                        aria-hidden="true"
-                        data-bs-slide="per"
-                        style="
-                          position: absolute;
-                          left: 40%;
+                          <div class="form_btn">
+                            <button
+                              [disabled]="i == 0"
+                              class="carousel-control-Previous"
+                              id="previous"
```

Comment:
what is use for this id and role and all ?

Reply:
data-slide="Previous"
 aria-hidden="true"
 data-bs-slide="per"

Reply:
Please check in all and remove if not required

Reply:
yes, i removed which are not required.